lib/app/modules/core/capability_sequence.rb

```diff
@@ -0,0 +1,122 @@
+module Inferno
+  module Sequence
+    class ConformanceSequence < SequenceBase
+
+      title 'Conformance Statement'
```

Comment:
Should this be capability statement?
```suggestion
      title 'Capability Statement'
```